### PR TITLE
1.3.0

### LINKED ISSUE #399: monocle-law holding us back
The most recently published version of monocle-law (1.2.2) is built against discipline 0.4, which in turn is built against specs2-core 3.6 and scalacheck 1.12.4, both of which are binary incompatible with everything recent.

discipline 0.5 (the version seen at the repository head) jumps to 3.7.3/1.13.1, but an even better target would be the current discipline 0.7, which is specs2 3.8.5 and scalacheck 1.13.2.

New artifacts would be greatly appreciated - thanks.

### PR DESCRIPTION
**Pull request by julien-truffaut**
Preparation for 1.3.0 release

fix #399

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -8,5 +8,5 @@ addSbtPlugin("com.typesafe.sbt"   % "sbt-site"             % "0.8.1")
 addSbtPlugin("net.virtual-void"   % "sbt-dependency-graph" % "0.7.4")
 addSbtPlugin("org.xerial.sbt"     % "sbt-sonatype"         % "0.5.0")
 addSbtPlugin("pl.project13.scala" % "sbt-jmh"              % "0.2.5")
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"          % "0.6.10")
+addSbtPlugin("org.scala-js"       % "sbt-scalajs"          % "0.6.12")
```
Comment (xuwei-k) [inline project/plugins.sbt]: latest scala-js version is 0.6.13

https://groups.google.com/d/msg/scala-internals/LGRU1gEwRgQ/7CAnITvBAQAJ

**Comment by paulp** [comment]
There's a macro-compat 1.1.1 - don't know if it matters. Otherwise looks great.

**Comment by paulp** [comment]
The linked messages about scalajs says "it contains some breaking changes in builds" so if the ambition for these artifacts is stability, a version which explicit declares breakage and which is barely hours old might not be the best bet.

**Comment by julien-truffaut** [comment]
scala.js breaking change seems to be around defaulting js backend to node. I guess that's what most people do in production anyway.

**Comment by julien-truffaut** [comment]
hum apparently `macro-compat` is producing some errors for scala 2.10

**Comment by aoiroaoino** [comment]
:+1: :+1: :+1:

**Pull request merged**